abs.yaml

```diff
@@ -0,0 +1,8 @@
+upload_channels:
```

Comment:
Reminder to remove before merge (although I am guessing you are already aware :))

Reply:
yes, I  am, but thank you !